src/furui/containers/local-timeline/index.tsx

```diff
@@ -29,6 +30,14 @@ export default () => {
     appStore.setIsUploading(false)
     setUploadState(0)
   }
+
+  useEffect(() => {
+    if (!inReplyTo) return
+    const post = appStore.posts.get(inReplyTo)
+    if (!post) return
```

Comment:
ないときは例外で良い気がする

Reply:
現状の mozuku では投稿が見えないとリプライは送ることができないはずなので、例外になっていたほうがわかりやすいんじゃないかと思った。ただ描写が止まるとつらいかもしれないので、`console.error` を出すでもいいかもしれない

```suggestion
    if (!post) {
      console.error(`replayToPost can not shown, failed to get post: ${inReplyTo}`)
    }
```